modules/@apostrophecms/schema/ui/apos/components/AposInputRelationship.vue

```diff
@@ -134,19 +134,19 @@ export default {
       }
     }
   },
+  mounted () {
+    this.validate(this.next);
```

Comment:
run a first validate otherwise we don't have the `limit reached` at first

Reply:
Yes agree, but actually I tested this way and it was working I had no required field complains when mounted:
![image](https://user-images.githubusercontent.com/17548370/165049513-960f5451-4f51-4985-baf4-a457b3d5b797.png)

But I suppose it is not working as expected.